manifests/b/Bitnami/TestLink/1.9.20-15/Bitnami.TestLink.locale.en-US.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Bitnami.TestLink
+PackageVersion: 1.9.20-15
+PackageLocale: en-US
+Publisher: Bitnami

```

Comment:
```suggestion
Publisher: Name of your company
```
I don't know if this is just me?
![image](https://user-images.githubusercontent.com/11600822/128933036-819fb32d-67c6-46af-9bf8-ca96527a6283.png)
![image](https://user-images.githubusercontent.com/11600822/128933087-912f507c-920b-4c5e-b6f0-2166f3e854cb.png)

Reply:
I let it finish successfully, `Name of your company` is the publisher.

![image](https://user-images.githubusercontent.com/52826317/128936035-cac7694d-9934-436b-869c-b066d0523386.png)